README.md

```diff
@@ -1,7 +1,300 @@
+
+Settings in the "more" panel:
+
+These are minimum filters that essentially act as prefilters prior to FDR calculation. The difference between setting them here versus in the "input" tab is that these parameters can then be part of the boosting grid search (see below).
+
+| Setting | Description| when to use                                                    |
+|---------|------------|----------------------------------------------------------------|
+| min. peptide fragments| minimum observed fragments per peptide | 0 by default, increase number in SDA searches                  |
+| min. coverage| number between 0 and 1. Minimum fraction of theoretical fragments required | included in boosting                                           |
+| min. coverage| number between 0 and 1. Minimum fraction of theoretical fragments required | included in boosting                                           |
+| min. peptide stubs| minimum number of fragment stubs observed | important for MS-cleavable crosslinkers, included in boosting  |
+|min. peptide doublets| minimum number of  peptide doublets observed | important for DSSO searches, included in boosting              |
+| boost separately|perform independent optimisation of each parameter| on by default                                                  |
+
+The settings in "define groups" are currently very beta and unsupported. They allow for splitting the dataset further down into custom groups for FDR calculation.
+
+#### Boosting
+xiFDR's boosting feature performs a grid search to optimise FDR settings at lower levels to reach the maximum number of matches passing validation at the desired FDR level. For example, enabling boosting for a residue pair-level FDR of 5% will tweak PSM, peptide pair and protein group level FDR cutoffs to maximise the number of residue pairs passing the 5% FDR threshold.
+
+Boosting is performed with a grid search of parameters as described in [Fisher et al. 2017](https://pubs.acs.org/doi/pdf/10.1021/acs.analchem.6b03745). 
+
+The user may control which parameters are part of boosting by changing the selection in the "boosting includes" button. 
+
+The "steps" controls how many steps of the grid search per parameter. The "between" box ensures that boosting is performed to maximise the number of heteromeric residue pairs/PPIs etc. passing FDR rather than the overall number. This is recommended for searches where the goal is to produce a protein-protein interaction network and where large numbers of heteromeric crosslinks are available.
```

Comment:
```suggestion
The "steps" controls how many steps of the grid search per round of optimization. The "between" box ensures that boosting is performed to maximise the number of heteromeric residue pairs/PPIs etc. passing FDR rather than the overall number. This is recommended for searches where the goal is to produce a protein-protein interaction network and where large numbers of heteromeric crosslinks are available.
```